cmd/lakectl/cmd/config/config.go

```diff
@@ -64,11 +79,16 @@ const (
 	HiveDBLocationURIKey = "metastore.hive.db_location_uri"
 
 	// Defaults
-	HiveDBLocationURI = "file:/user/hive/warehouse/"
+	HiveDBLocationURI        = "file:/user/hive/warehouse/"
+	ConfigServerEndpointURL  = "server.endpoint_url"
+	DefaultServerEndpointURL = "http://127.0.0.1:8000"
+	ConfigAccessKeyID        = "credentials.access_key_id"
+	ConfigSecretAccessKey    = "credentials.secret_access_key"
```

Comment:
Note that at least the comments want to split it up into Keys and Defaults -- which doesn't happen here.  I don't particularly care, but if we don't do it then let's remove the dead comments and be consistent about the keys and defaults.